astropy/cosmology/io/_parameter.py

```diff
@@ -0,0 +1,74 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+
+from astropy.cosmology.parameter import Parameter
+from astropy.table import Column
+from astropy.modeling import Parameter as ModelParameter
+
+
+def _convert_Parameter_to_Column(parameter, value, param_meta=None):
+    """Convert a Cosmology Parameter to a Table Column.
+
+    Parameters
+    ----------
+    parameter : `astropy.cosmology.parameter.Parameter`
+    value : Any
+    meta : dict or None, optional
```

Comment:
Name here and in signature are different (I think I'd use `meta`)

Reply:
Agreed.

Reply:
Done.